//package io.core.protocol
import "RankPlayer.proto";
message DseRankList
{
	optional int32	type 				= 1;	//1演武场
	optional RankPlayerList ranklist 	= 2;	//排行
}
